quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/integration/jaxrs/QuarkusKeycloakApplication.java

```diff
@@ -85,23 +80,23 @@ protected void createTemporaryAdmin(KeycloakSession session) {
 
         try {
             //Integer expiration = Configuration.getOptionalKcValue(BootstrapAdminOptions.EXPIRATION.getKey()).map(Integer::valueOf).orElse(null);
-            if (StringUtil.isNotBlank(adminPassword)) {
-                createTemporaryMasterRealmAdminUser(adminUsername, adminPassword, /*expiration,*/ session);
+            if (StringUtil.isNotBlank(adminPassword) && !createTemporaryMasterRealmAdminUser(adminUsername, adminPassword, /*expiration,*/ session)) {
+                throw new RuntimeException("Aborting startup and the creation of the master realm, because the temp admin user account could not be created.");
```

Comment:
```suggestion
                throw new RuntimeException("Aborting startup and the creation of the master realm, because the temporary admin user account could not be created.");
```
might be better, right?

Reply:
Ok, applied.